packages-ts/gauntlet-terra-contracts/src/lib/contracts.ts

```diff
@@ -30,36 +31,27 @@ export type Contract = {
 
 export type Contracts = Record<CONTRACT_LIST, Contract>
 
-export const loadContracts = (): Contracts => {
+export const loadContracts = (version): Contracts => {
   return Object.values(CONTRACT_LIST).reduce((agg, id) => {
     return {
       ...agg,
       ...{
         [id]: {
           id,
           abi: getContractABI(id),
-          bytecode: getContractCode(id),
+          bytecode: getContractCode(id, version),
         },
       },
     }
   }, {} as Contracts)
 }
 
-// TODO: Pull it from a Github versioned release artifact
-export const getContractCode = (contractId: CONTRACT_LIST): string => {
-  // Possible paths depending on how/where gauntlet is being executed
-  const possibleContractPaths = [
-    path.join(__dirname, '../..', './artifacts/bin'),
-    path.join(process.cwd(), './artifacts/bin'),
-    path.join(process.cwd(), './packages-ts/gauntlet-terra-contracts/artifacts/bin'),
-  ]
```

Comment:
We should still leave this option open while in the development of the next contract/gauntlet version, right @RodrigoAD? We could also use an explicit version like `next` or `local` to configure this mode.

Reply:
Let's also make sure we use the same artifact loading workflow on Solana.

Reply:
Yeah good point. Let's leave the option with `local` mode.

We could do this exact change next on Solana